Makefile

```diff
@@ -127,7 +133,7 @@ requirements: virtualenv $(REQUIREMENTS)
 	@echo
 	for req in $(REQUIREMENTS); do \
 		echo "Installing $$req..." ; \
-		. $(VIRTUALENV_DIR)/bin/activate && pip install -U -q -r $$req ; \
+		. $(VIRTUALENV_DIR)/bin/activate && pip install $(PIP_OPTIONS) $$req ; \
```

Comment:
We always want to specify at least `-r` since we are installing from a file, right?